tests/rptest/transactions/verifiers/stream_verifier.py

```diff
@@ -0,0 +1,202 @@
+            return False
+
+    def ensure_progress(self, delta, timeout_sec):
+        if not self.is_alive(self._node):
+            self.raise_on_violation(self._node)
+            raise CrushedException(
+                f"{title} is crashed. Check its logs for details")
+        # TODO: Check number of processes messages and ensure it changed
+
+    # Service overrides
+    def start_node(self, node, timeout_sec=10):
+
+        # Build cmd line
+        cmd = "python3 <remote>/stream_verifier_txn.py"
+        # run webserver
+        node.account.ssh(cmd)
+
+        wait_until(lambda: self.is_alive(node),
+                   timeout_sec=timeout_sec,
+                   backoff_sec=1,
+                   err_msg=f"{title} service {node.account.hostname} "
+                   f"failed to start within {timeout_sec} sec",
+                   retry_on_exc=False)
+        self._node = node
+        wait_until(lambda: self.is_ready(),
+                   timeout_sec=timeout_sec,
+                   backoff_sec=1,
+                   err_msg=f"{title} service {node.account.hostname} failed "
+                   f"to become ready within {timeout_sec} sec",
+                   retry_on_exc=False)
+
+    def stop_node(self, node):
+        # Send termina
+        node.account.ssh("bash /opt/remote/control/stop.sh streamverifier")
+
+    def clean_node(self, node):
+        pass
+
+    def wait_node(self, node, timeout_sec=sys.maxsize):
+        wait_until(lambda: not (self.is_alive(node)),
+                   timeout_sec=timeout_sec,
+                   backoff_sec=1,
+                   err_msg=f"{title} service {node.account.hostname} failed "
+                   f"to stop within {timeout_sec} sec",
+                   retry_on_exc=False)
+        return True
+
+    ### RPCs
+    @staticmethod
+    def _raise_on_non_success(request_result):
+        if request_result.status_code != 200:
+            raise Exception(
+                f"unexpected status code: {request_result.status_code} "
+                f"content: {request_result.content}")
+
+    def _get_ip(self):
+        return self._node.account.hostname
+
+    def check_status(self):
+        r = requests.get(f"http://{self._get_ip()}:8090/status")
+        self._raise_on_non_success(r)
+        return r.json()
+
+    def remote_start_produce(self,
+                             connection,
+                             topic,
+                             partitions,
+                             key_set_cardinality=10,
+                             abort_probability=0.3):
+        self._partitions = partitions
+
+        r = requests.post(f"http://{self._get_ip()}:8080/produce",
+                          json={
+                              "brokers": connection,
+                          })
+        self._raise_on_non_success(r)
+
+    def remote_start_consume(self):
+        r = requests.post(f"http://{self._get_ip()}:8080/consume", json={})
```

Comment:
given these request handling is not included in this PR, its a bit odd IMO to have the post methods here.

Reply:
Handling is not present here yet, but will be.